packages/worker/src/api/routes/global/tests/auth.spec.ts

```diff
@@ -292,9 +292,9 @@ describe("/api/global/auth", () => {
       it("redirects to auth provider", async () => {
         nock("http://someconfigurl").get("/").times(1).reply(200, {
           issuer: "test",
-          authorization_endpoint: "http://localhost/auth",
-          token_endpoint: "http://localhost/token",
-          userinfo_endpoint: "http://localhost/userinfo",
+          authorization_endpoint: "http://example.com/auth",
```

Comment:
Using `localhost` here now that `nock` is in use made the tests fail because we allow legit connections to `localhost` in order to make requests to containers, so I changed it to `example.com`.